azkaban/remote.py

```diff
@@ -655,14 +661,19 @@ def _refresh(self, password=None):
     self._logger.info('Refreshed.')
 
   def _run_options(self, name, flow, jobs=None, concurrent=True,
-    properties=None, on_failure='finish', notify_early=False, emails=None):
+                   properties=None, on_failure='finish', notify_early=False,
+                   emails=None, disabled_jobs=None):
     """Construct data dict for run related actions.
 
     See :meth:`run_workflow` for parameter documentation.
 
     """
     if not jobs:
-      disabled = '[]'
+      if not disabled_jobs:
+        disabled = '[]'
+      else:
+        disabled = "[{0}]".format(",".join(['"{0}"'.format(x)
```

Comment:
Indentation and use `%` instead of `.format` (consistency with the rest of the code). Also, it'd be nice to add a check that at most one of `jobs` and `disabled_jobs` is defined.